File: openapi-generator/kotlin-spring/src/main/kotlin/org/openapitools/model/HosLogsResponseLogs.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * 
 * @param locLng Longitude at which the log was recorded.
 * @param logStartMs The time at which the log/HOS status started in UNIX milliseconds.
 * @param driverId ID of the driver.
 * @param statusType The Hours of Service status type. One of `OFF_DUTY`, `SLEEPER_BED`, `DRIVING`, `ON_DUTY`, `YARD_MOVE`, `PERSONAL_CONVEYANCE`.
 * @param locCity City in which the log was recorded.
 * @param groupId ID of the group.
 * @param locName Name of location at which the log was recorded.
 * @param locLat Latitude at which the log was recorded.
 * @param remark Remark associated with the log entry.
 * @param locState State in which the log was recorded.
 * @param vehicleId ID of the vehicle.
 * @param codriverIds 
 */
data class HosLogsResponseLogs (

        @JsonProperty("locLng") val locLng: Double? = null,

        @JsonProperty("logStartMs") val logStartMs: Long? = null,

        @JsonProperty("driverId") val driverId: Long? = null,

        @JsonProperty("statusType") val statusType: String? = null,

        @JsonProperty("locCity") val locCity: String? = null,

        @JsonProperty("groupId") val groupId: Long? = null,

        @JsonProperty("locName") val locName: String? = null,

        @JsonProperty("locLat") val locLat: Double? = null,

        @JsonProperty("remark") val remark: String? = null,

        @JsonProperty("locState") val locState: String? = null,

        @JsonProperty("vehicleId") val vehicleId: Long? = null,

        @JsonProperty("codriverIds") val codriverIds: List<Long>? = null
) {

}

File: openapi-generator/scala-gatling/src/gatling/scala/org/openapitools/client/model/HosLogsResponseLogs.scala

package org.openapitools.client.model


case class HosLogsResponseLogs (
    /* Longitude at which the log was recorded. */
    _locLng: Option[Double],
    /* The time at which the log/HOS status started in UNIX milliseconds. */
    _logStartMs: Option[Long],
    /* ID of the driver. */
    _driverId: Option[Long],
    /* The Hours of Service status type. One of `OFF_DUTY`, `SLEEPER_BED`, `DRIVING`, `ON_DUTY`, `YARD_MOVE`, `PERSONAL_CONVEYANCE`. */
    _statusType: Option[String],
    /* City in which the log was recorded. */
    _locCity: Option[String],
    /* ID of the group. */
    _groupId: Option[Long],
    /* Name of location at which the log was recorded. */
    _locName: Option[String],
    /* Latitude at which the log was recorded. */
    _locLat: Option[Double],
    /* Remark associated with the log entry. */
    _remark: Option[String],
    /* State in which the log was recorded. */
    _locState: Option[String],
    /* ID of the vehicle. */
    _vehicleId: Option[Long],
    _codriverIds: Option[List[Long]]
)
object HosLogsResponseLogs {
    def toStringBody(var_locLng: Object, var_logStartMs: Object, var_driverId: Object, var_statusType: Object, var_locCity: Object, var_groupId: Object, var_locName: Object, var_locLat: Object, var_remark: Object, var_locState: Object, var_vehicleId: Object, var_codriverIds: Object) =
        s"""
        | {
        | "locLng":$var_locLng,"logStartMs":$var_logStartMs,"driverId":$var_driverId,"statusType":$var_statusType,"locCity":$var_locCity,"groupId":$var_groupId,"locName":$var_locName,"locLat":$var_locLat,"remark":$var_remark,"locState":$var_locState,"vehicleId":$var_vehicleId,"codriverIds":$var_codriverIds
        | }
        """.stripMargin
}
